### ISSUE TITLE
c10m5_houseboat has 4 missing ladder navs at the boathouse

### ISSUE THREAD

**Issue by alexiscoutinho** [Tsuey/L4D2-Community-Update#60]
This seems to be a recurring issue.

![Untitled](https://user-images.githubusercontent.com/26471497/133505310-411672e8-fe0e-4e2c-95fa-12557e7af6a1.png)

**Comment by Nesciuse** [comment]
I don't think this is an issue. These ladders are just meant for players in versus

**Comment by alexiscoutinho** [comment]
Hmmm, but without them the infected can't properly/fully pathfind to the balcony and roof to punish boosted survivors. Plus, the infected look dumb when they take detours, especially when the player knows there is a ladder there from experience. If the ladders make sense in versus, they would make the Director more realistic.
